keras/engine/base_layer_test.py

```diff
@@ -1116,6 +1117,33 @@ def __init__(self, var1, var2, var3=None, **kwargs):
         with self.assertRaises(NotImplementedError):
             config = layer.get_config()
 
+    def test_model_compute_output_signature_dtype_single_output(self):
+        with policy_scope("mixed_float16"):
```

Comment:
Is there anything to do with mixed_precision here? or it can be removed here in this test.